addons/armaos/functions/fnc_terminal_updatePromptPointer.sqf

```diff
@@ -0,0 +1,8 @@
+params ["_computer"];
+
+private _pointer = _computer getVariable "AE3_filepointer";
+private _terminal = _computer getVariable "AE3_terminal";
+
+_terminal set ["AE3_terminalPrompt", "/" + (_pointer joinString "/") + ">"];
+
+_computer setVariable ["AE3_terminal", _terminal];
```

Comment:
Should be unnecessary, because variables referencing hashmaps are only pointers to the actual hashmap object, not the hashmap itself.

Reply:
removed in https://github.com/y0014984/Advanced-Equipment/pull/90/commits/996717f1cedd397770d94ec876aedb11e65687ad